src/ast/stmt/ConstStmt.php

```diff
@@ -21,55 +21,57 @@
  */
 namespace QuackCompiler\Ast\Stmt;
 
+use \QuackCompiler\Intl\Localization;
 use \QuackCompiler\Parser\Parser;
+use \QuackCompiler\Scope\Kind;
 use \QuackCompiler\Scope\Meta;
+use \QuackCompiler\Types\TypeError;
 
 class ConstStmt extends Stmt
 {
-    public $definitions;
-    private $scoperef;
+    public $name;
+    public $type;
+    public $value;
 
-    public function __construct($definitions)
+    public function __construct($name, $type, $value)
     {
-        $this->definitions = $definitions;
+        $this->name = $name;
+        $this->type = $type;
+        $this->value = $value;
     }
 
     public function format(Parser $parser)
     {
-        $source = 'const ';
-        $first = true;
+        $source = 'const ' . $this->name;
 
-        foreach ($this->definitions as $def) {
-            if (!$first) {
-                $source .= $parser->indent();
-                $source .= '    , ';
-            } else {
-                $first = false;
-            }
-
-            $source .= $def[0];
-            $source .= ' :- ';
-            $source .= $def[1]->format($parser);
-            $source .= PHP_EOL;
+        if (!is_null($this->type)) {
+            $source .= ' :: ' . $this->type;
         }
 
+        $source .= ' :- ' . $this->value->format($parser);
```

Comment:
Almost there! Because the expression can be really big, but we can get the value this way!